presto-main/src/test/java/com/facebook/presto/operator/project/TestPageProcessor.java

```diff
@@ -121,8 +125,7 @@ public void testPartialFilter()
 
         Page inputPage = new Page(createLongSequenceBlock(0, 100));
 
-        PageProcessorOutput output = pageProcessor.process(SESSION, new DriverYieldSignal(), inputPage);
-        assertEquals(output.getRetainedSizeInBytes(), inputPage.getRetainedSizeInBytes());
+        Iterator<Optional<Page>> output = processAndAssertRetainedPageSize(pageProcessor, inputPage);
```

Comment:
I debugged the test `testPartialFilter` and in `processAndAssertRetainedPageSize` method we assert that `assertEquals(memoryContext.getBytes(), 0);`. However, when I look at what's being retained by the `output` instance in that method there is plenty of stuff. So, I am confused. This also ties back to my previous question about what we are tracking with these memory contexts.